packages/Rakefile

```
@@ -163,6 +163,27 @@ class GroongaPackageTask < PackagesGroongaOrgPackageTask
   def built_package_n_split_components
     3
   end
+
+  def download_packages(target_namespace)
+    return if target_namespace == :windows
+    super
+  end
+
+  def create_htaccess(target_namespace)
+    return unless target_namespace == :windows
+
+    repositories_dir = download_repositories_dir(target_namespace)
+    mkdir_p(repositories_dir)
+    htaccess_path = "#{repositories_dir}/.htaccess"
+    rm_rf(htaccess_path)
+    
+    __send__("#{target_namespace}_targets").each do |target|
+      redirect_url = built_package_url(target_namespace, target)
+      redirect_target = target.gsub(@version, "latest")
+      htaccesss_value = "RewriteRule #{redirect_target} #{redirect_url}\n"
+      File.write(htaccess_path, htaccesss_value, mode: "a+")  
+    end
```

How about opening a file instead of calling `File.write()` multiple times? I haven't seen the usage (Use `FileUtils.rm_f` and multiple `File.write(mode: "a+")` to create multi lines content).

```suggestion
    File.open(htaccess_path, "w") do |htaccess|
      __send__("#{target_namespace}_targets").each do |target|
        redirect_url = built_package_url(target_namespace, target)
        redirect_target = target.gsub(@version, "latest")
        htaccess.puts("RewriteRule #{redirect_target} #{redirect_url}")
      end
    end
  end
```

---

I have fixed it.